src/client/common.py

```diff
@@ -1,19 +1,16 @@
-import base64
-import time
-import hmac
-import os
-import hashlib
-import six
-import sys
+import base64, time, hmac, os, hashlib
```

Comment:
trivial comment, this a less widely followed style for importing, i would prefer sticking with the former one.